core/src/mindustry/mod/ContentParser.java

```diff
@@ -870,6 +870,15 @@ private GenericMesh parseMesh(Planet planet, JsonValue data){
 
         String tname = Strings.capitalize(data.getString("type", "NoiseMesh"));
 
+        Color[] colors = new Color[0];
+        if(tname.equals("SunMesh")){
```

Comment:
You can do this inside the SunMesh case branch with braces and `yield new SunMesh(...) at the end`.

Reply:
So that was a thing? 👀